src/components/_tests/ContextBar.test.js

```diff
@@ -0,0 +1,561 @@
+import '@testing-library/jest-dom'
+
+import { fireEvent, render, screen } from '@testing-library/react'
+import { div, h } from 'react-hyperscript-helpers'
+import { tools } from 'src/components/notebook-utils'
+import { MenuTrigger } from 'src/components/PopupTrigger'
+import { Ajax } from 'src/libs/ajax'
+import { CloudEnvironmentModal } from 'src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal'
+
+import { ContextBar } from '../ContextBar'
```

Comment:
dont use relative imports

Reply:
I would instead push to move this test file to be co-located next to the code-under-test... I think this was discussed a bit in the UIWG quite a few weeks ago... but my memory is fuzzy.  We may not have gotten a hard consensus on it.

Reply:
You're correct - and Justin's comment reminded me to move the test!